### ISSUE TITLE
Allow overwriting include

### ISSUE THREAD

**Issue by leopf** [syumai/dejs#72]
To allow for a more dynamic resolution of partial templates I suggest allowing overwriting the include function with a param named "include". This is currently not possible because of the order in which the args object is constructed.

I suggest swapping lines 71 and 72 to allow for this to be possible.

The current args object:
```ts
const args = {
    ...params,
    include,
    $$OUTPUT: output,
    $$FINISHED: resolve,
    $$ESCAPE: escape
};
```

The suggested args object:
```ts
const args = {
    include,
    ...params,
    $$OUTPUT: output,
    $$FINISHED: resolve,
    $$ESCAPE: escape
};
```

**Comment by syumai** [comment]
This change looks OK, I'll implement this feature.

**Comment by syumai** [comment]
This feature was released at 0.10.1
